rotkehlchen/constants/timing.py

```diff
@@ -21,3 +21,4 @@
 ENS_UPDATE_INTERVAL = DAY_IN_SECONDS
 
 ETH_PROTOCOLS_CACHE_REFRESH = DAY_IN_SECONDS * 3
+DATA_UPDATES_REDETECT = DAY_IN_SECONDS
```

Comment:
As mentioned to Alexey in the other PR let's keep common naming across such variables

```suggestion
DATA_UPDATES_REFRESH = DAY_IN_SECONDS
```